stories/2-Card.stories.js

```diff
@@ -2,20 +2,21 @@ import React from "react";
 import Card from "../src/components/Card";
 import Button, {InputButton} from "../src/styles/Button";
 import {PrimaryWithText} from "./1-Button.stories";
-import {CardPadding, ContextStyle, HintStyle} from "../src/styles/Card";
+import {CardPadding, ContextStyle, HintStyle, OnBoardStyle} from "../src/styles/Card";
```

Comment:
I see a pattern emerging and I like it the use of specific style components to extend flexibility.